packages/xstate-react/test/useActor.test.tsx

```diff
@@ -511,4 +512,53 @@ describeEachReactMode('useActor (%s)', ({ render, suiteKey }) => {
 
     expect(spy).not.toBeCalled();
   });
+
+  it('should work with a null actor', () => {
+    const Child = (props: {
+      actor: ActorRef<any, { count: number }> | undefined;
+    }) => {
+      const [state] = useActor(props.actor ?? createNullActor());
+
+      try {
+        // @ts-expect-error (state can be undefined)
+        state.count;
+      } catch (_) {}
+
+      if (state) {
+        // another type test... probably a better way to do this
+        ((_count: number) => void 0)(state.count);
+      }
```

Comment:
I'd love to integrate a better way for writing type tests but so far we have been using things like this:
```suggestion
      // @ts-expect-error
      ((_accept: { count: number }) => {})(state);
      ((_accept: { count: number } | undefined) => {})(state);
```